util/Minimization_Monitor/image_gen/ush/MinMon_Plt.sh

```diff
@@ -45,38 +45,27 @@ do
    shift
 done
 
+if [[ ${#MINMON_SUFFIX} -le 0 ]]; then 
+   echo "No suffix supplied, unable to proceed"
+   exit 2
+fi
+
 if [[ ${#RUN} -le 0 ]]; then 
    export RUN=gdas
 fi
+run_suffix=${MINMON_SUFFIX}_${RUN}
 
 echo "MINMON_SUFFIX = $MINMON_SUFFIX"
 echo "PDATE         = $PDATE"
 echo "RUN           = $RUN"
 
 
-
-if [[ ${#RUN} -gt 0 ]]; then 
-   run_suffix=${MINMON_SUFFIX}_${RUN}
-else
-   run_suffix=${MINMON_SUFFIX}
-fi
-
+#----------------------------------------
+# source config, and user_settings files
+#----------------------------------------
 this_dir=`dirname $0`
-
-#--------------------------------------------------
-# source verison, config, and user_settings files
-#--------------------------------------------------
 top_parm=${this_dir}/../../parm
 
-minmon_version_file=${minmon_version:-${top_parm}/MinMon.ver}
-if [[ -s ${minmon_version_file} ]]; then
-   . ${minmon_version_file}
-   echo "able to source ${minmon_version_file}"
-else
-   echo "Unable to source ${minmon_version_file} file"
-   exit 2
-fi
-
 minmon_config=${minmon_config:-${top_parm}/MinMon_config}
 if [[ -s ${minmon_config} ]]; then
    . ${minmon_config}
```

Comment:
Adding a check that the `$minmon_config` and `$minmon_user_setting` were sourced correctly (i.e. `$? = 0`) would make debugging easier.

Reply:
Good suggestion.  I've made that change.